SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestClientNetOwnership/SpatialTestNetOwnership.cpp

```diff
@@ -67,8 +83,10 @@ void ASpatialTestNetOwnership::PrepareTest()
 
 	// Server 1 spawns the NetOwnershipCube and registers it for auto-destroy.
 	AddStep(TEXT("SpatialTestNetOwnershipServerSpawnCube"), FWorkerDefinition::Server(1), nullptr, [this]() {
+		// The position is chosen as a hack to make sure the cube spawns on Server 1's turf, so we don't run into issues with the framework
+		// itself...
 		ANetOwnershipCube* Cube =
-			GetWorld()->SpawnActor<ANetOwnershipCube>(FVector::ZeroVector, FRotator::ZeroRotator, FActorSpawnParameters());
+			GetWorld()->SpawnActor<ANetOwnershipCube>(FVector(-50.0f, -50.0f, 0.0f), FRotator::ZeroRotator, FActorSpawnParameters());
```

Comment:
Could use `GetWorkerEntityPosition` here

Reply:
Yeah... but I don't like this entire situation. I guess ideally the test could be reworked so it uses the debug interface instead? That way we won't have to worry about things migrating at unfortunate times.

Reply:
I could add it to the ticket to fix this test, to consider reworking it to use the debug interface.

Reply:
Sounds good